lib/library/books/book.ex

```diff
@@ -17,8 +17,9 @@ defmodule Library.Books.Book do
     field :title, :string
     field :type, :string
     many_to_many :author, Library.Books.Author, join_through: "author_books"
-    has_one :book_loan, Library.Books.BookLoan
+    has_many :book_loan, Library.Books.BookLoan
```

Comment:
Should this be `has_many` or stay as `has_one`? Would each book be stored as it's own entry in the books table, even if it's a duplicate title? If so I would think the book could only have one loan at a time..?